content/teams/DemoTeam/The_Coding_Hush.md

```diff
@@ -0,0 +1,134 @@
+                           abled is a complex phenomenon which reflects an interaction between the features of
+                           a person�s body and the features of the society in which he or she lives. To 
+                           overcome this, the use of computer science should undertake a progressive path 
+                           which we aim to achieve in this attempt of making their lives easier.
+	                   The main ideology behind this project is to provide a helping hand to the
+                           differently abled in a more conservative way. This is a web application named
+                           �ASSISTIVE TOUCH� where we have created a room of solutions for people suffering
+                           from various ailments of the physical as well as the mental kind. Following the 
+                           logon, they can avail the comfort of sharing their grievances on a platform that�s
+                           user friendly and open to all. Every candidate is given the privilege of creating 
+                           their own user identity which is password protected and highly encrypted. The 
+                           webpage is categorized in numerous ways for people who have lost their sense of
+                           hearing, vision, motility, limbs as well for people who have emotional or 
+                           psychological ailments.  The amenities that the web application provides include
+                           information about the various helping centres, government schemes, opportunities 
+                           for employment as well as special schooling, various informative articles, podcasts,
+                           links for video playlists, a network for connecting them with people who share the
+                           same grievances, calming music playlists and so on.
+	                   The web application also provides a discussion forum that allows them to share their
+                           thoughts and experiences as well as read others� experiences as well. Their also 
+                           exists a LiveChat and a HelpDesk section. A feedback section will also be provided 
+                           for constructive criticism and for the betterment of the web application. They will 
+                           also be able to contact us through the function made available.
+	                   We might not have the experience or the pain they go through on a daily basis but 
+                           we do understand it and hence, this web application is our way of empathizing with
+                           them.
+
+
+
+
+_**Tech-Stack-Used**_ : Python, Java, Html5, CSS3, flask, React Js, php, javascript, sql lite3, MySql, 
+                         Oracle, sql server 
+
+_**GitHub-Link**_ :   Github project Repository link Goes here (make a blank repository). 

```

Comment:
Mention your hackathon project blank repository here. ( Link required )

check listing not done.